mono_repo/lib/src/commands/travis.dart

```diff
@@ -312,6 +323,18 @@ esac
 ''';
 }
 
+String _safeEcho(bool prettyAnsi, AnsiCode code, String value) {
+  // Safely escape everything:
+  // 1 - use single quotes.
+  // 2 - if there is a single quote in the string
+  //     2.1 end the before the single quote
+  //     2.2 echo the single quote escaped
+  //     2.3 continue the string
+  // See https://stackoverflow.com/a/20053121/39827
+  value = value.replaceAll("'", "'\\''");
```

Comment:
are there any tests that exercise this behavior?

Reply:
now there are